### DISCUSSION TITLE
Size Limit

### DISCUSSION THREAD

**Discussion by mikelawell** [janisdd/vscode-edit-csv#98] [Q&A]
There currently is a limit of 50 MB. Is there a way to override? I'm editing files as large as a GB.

I looked in the types.d.ts for a config but didn't see one.

**Comment by janisdd** [comment]
There are some settings from vs code like `Large File Optimizations` and `Max Memory For Large Files MB` that could be useful or you could try the extension  https://marketplace.visualstudio.com/items?itemName=mbehr1.vsc-lfs

There is actually no specific code in the extension preventing large files support, only the check

```ts
if (!vscode.window.activeTextEditor || !isCsvFile(vscode.window.activeTextEditor.document)) {
  vscode.window.showInformationMessage("Open a csv file first to show the csv editor or file too large (> 50MB)")
  return false
}
```
So vs code don't return an `activeTextEditor` or `document` at some file size limit...

However, if some of the ideas above works, I wouldn't recommend using this extension with such large files.
I normally test with a ~12 MB file and it lags noticeably...

Maybe another extension can handle such large files or maybe another editor?

**Comment by mikelawell** [comment]
Thank you
